inc/task.class.php

```diff
@@ -750,11 +757,14 @@ public function getAgentsFromActors($actors = [], $use_cache = false) {
              * TODO: The following should be replaced with Dynamic groups
              */
             case 'PluginFusioninventoryAgent':
-               switch($itemid) {
+               switch ($itemid) {
+
                   case "dynamic":
```

Comment:
I supposed that "dynamic" and "dynamic-same-subnet" should be safely removed right ?
In this case you can also remove the switch/case